src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -643,14 +643,14 @@ private CompareExchangeTombstonesCleanupState GetMaxCompareExchangeTombstonesEta
                         return CompareExchangeTombstonesCleanupState.InvalidDatabaseObservationState;
 
                     var tryGetValue = nodeReport.Report.TryGetValue(state.Name, out var report);
-                    Debug.Assert(tryGetValue, $"Could not find state for node '{nodeTag}' for database '{state.Name}'.");
+                    Debug.Assert(tryGetValue, $"Could not find report for node '{nodeTag}' for database '{state.Name}'.");
                     if (tryGetValue == false)
                     {
                         return CompareExchangeTombstonesCleanupState.InvalidDatabaseObservationState;
                     }
 
                     var clusterWideTransactionIndex = report.LastClusterWideTransactionRaftIndex;
-                    if (clusterWideTransactionIndex < maxEtag)
+                    if (clusterWideTransactionIndex > maxEtag)
```

Comment:
oh, does it mean that this never worked properly?

Reply:
I changed it back.
It was correct in the first place.